test/functionalTest/cases/3004_v2_reg_create/v2_reg_create_v1_forward.test

```diff
@@ -0,0 +1,181 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Create registration using APIv2 and provoke v1 forwards
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
```

Comment:
Maybe a similar test using query instead of update would be added? In a separated .test, for clarity.

Reply:
Not sure what you mean ...

Step 03 does a GET, (a query) forwarded to CP1
Step 04 does an UPDATE, forwarded to CP1

Then step 05 makes sure that step 04 worked correctly.
I believe that is complete already ... Right?

Reply:
Yes. I missed step 4, sorry. NTC.